mage.go

```diff
@@ -1,5 +1,3 @@
-// +build ignore
```

Comment:
I'm not sure why this was removed. My understanding, is that this sort of get's used as a convention, so that this file gets ignore my the build process and is only considered when running `go run mage.go` which will ignore the tag.

Reply:
I was just thinking about commenting on it - I ended up removing it because it was not letting me `go mod vendor` the mage [CLI](github.com/magefile/mage/mage) package dependency and failing at the test step with:
```
$ go run mage.go build:satellite
...
=====> Running GRPC Codegen inside docker...

mage.go:21:2: cannot find package "github.com/magefile/mage/mage" in any of:
	/go/src/github.com/gravitational/satellite/vendor/github.com/magefile/mage/mage (vendor tree)
	/usr/local/go/src/github.com/magefile/mage/mage (from $GOROOT)
	/go/src/github.com/magefile/mage/mage (from $GOPATH)
Error: running "docker run --volume=/home/deemok/go/src/github.com/gravitational/satellite:/go/src/github.com/gravitational/satellite:delegated satellite-grpc-buildbox:7.1.0-21-gb1ece2e9-dirty sh -c cd /go/src/github.com/gravitational/satellite/ && go run mage.go internal:grpcAgent internal:grpcDebug" failed with exit code 1
exit status 1
```
You can see the effect by undoing the change and running `go mod vendor` which then deletes the dependency:
```
modified:   mage.go
deleted:    vendor/github.com/magefile/mage/internal/run.go
deleted:    vendor/github.com/magefile/mage/mage/command_string.go
deleted:    vendor/github.com/magefile/mage/mage/magefile_tmpl.go
deleted:    vendor/github.com/magefile/mage/mage/main.go
deleted:    vendor/github.com/magefile/mage/mage/template.go
deleted:    vendor/github.com/magefile/mage/parse/parse.go
modified:   vendor/modules.txt
```
I'm not sure how to go about it yet as I cannot quite explain the behavior of `go mod` in this context.

Reply:
It looks like it only "handles" `ignore` tag in this way - if I use another one (i.e. `mage`), it does not happen - I'll change to use the new tag.